Solutions/ServiceManifests/OperationsServiceManifest.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Is the intention that this stuff will eventually move into the individual service projects?

Reply:
Yes. They are there for experimentation at the moment. I can always delete them later if it's confusing.